src/evergreen_mcp/server.py

```diff
@@ -1,31 +1,42 @@
-"""MCP server for Evergreen"""
+"""FastMCP server for Evergreen
+
+This module provides the main MCP server using FastMCP framework.
+It handles server lifecycle, configuration, and tool registration.
+"""
 
 import argparse
-import json
 import logging
 import os
 import os.path
 import sys
-from asyncio import run
-from collections.abc import AsyncIterator, Sequence
 from contextlib import asynccontextmanager
+from dataclasses import dataclass
+from typing import AsyncIterator
 
-import mcp.server.stdio
-import mcp.types as types
 import yaml
-from mcp.server.lowlevel import NotificationOptions, Server
-from mcp.server.models import InitializationOptions
+from fastmcp import Context, FastMCP
+
+# Add src directory to path for imports when running directly
+_src_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _src_dir not in sys.path:
+    sys.path.insert(0, _src_dir)
```

Comment:
I think this shouldn't be running if installed under `pip install -e .` or whatever